testing/yuca/ca/test_cca.py

```diff
@@ -30,6 +30,20 @@ def test_get_frame(self):
             elif True in (frame_mode == np.array([4])):
                 self.assertEqual(4, len(frame_return))
             
+    def test_id_conv(self):
```

Comment:
This tests to make sure the identity layer actually performs and identity convolution. 
Note that the purpose of using `id_layer` instead of the grid itself is to provide flexibility for larger "inner neighborhoods," such as in the Larger-than-Life or SmoothLife frameworks.